classes/admin/class-mondu-admin-form-fields.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
```

Comment:
Question

Since we have 

`if ( ! defined( 'ABSPATH' ) ) {
    exit;
}`

in every file, shouldn't we have it here as well?

also we don't have a namespace here, but we declare a FormFields class, which can cause problems I believe.